toolbox/gui/figure_3d.m

```diff
@@ -1247,7 +1247,7 @@ function SetStandardView(hFig, viewNames)
     % Displaying a surface: Load the SCS field from the MRI
     if isempty(Ranat) && ~isempty(TessInfo) && ~isempty(TessInfo(1).SurfaceFile)
         % Get subject
-        sSubject = bst_get('SurfaceFile', TessInfo(1).SurfaceFile);
+        sSubject = db_get('SurfaceFile', TessInfo(1).SurfaceFile);

```

Comment:
You should either stick with bst_get() here or refactor the rest of this function. See the following line: it requires legacy fields sSubject.Anatomy.

Reply:
sSubject.Anatomy is still used in the following lines...

Reply:
Solved in 3834c6560c3d94138f25784b7f976ee4029b297a